templates/bin/fixity-cron

```diff
@@ -6,7 +6,7 @@ EMAILS=$(sqlite3 /var/archivematica/storage-service/storage.db 'select email fro
 
 function run_fixity() {
 echo "Fixity scan starts at $HOSTNAME" | ts '[%Y-%m-%d %H:%M:%S %Z]' > $REPORTS/fixity-$START_DATE.txt
-/usr/local/bin/fixity scanall 2>&1 |ts '[%Y-%m-%d %H:%M:%S %Z]' >> $REPORTS/fixity-$START_DATE.txt
+/usr/bin/nice -n -19 /usr/local/bin/fixity scanall 2>&1 |ts '[%Y-%m-%d %H:%M:%S %Z]' >> $REPORTS/fixity-$START_DATE.txt
```

Comment:
Is this here to reduce the impact of fixity in the system where it's running? And have you confirmed that it's actually a problem? AFAIK fixity is I/O-bound and all the heavy lifting is done by Storage Service.

Reply:
You are right, I'm discarding that commit.